src/components/api-reference/scoring.js

```diff
@@ -1,28 +1,1004 @@
+          notes: 'Skipped measurements may still contribute to the score by bonuses'
+        },
+        {
+          name: 'totalMeasurementPoints',
+          value: 'number',
+          description: 'Total points the measurement can contribute including base and bonus',
+          notes: ''
+        },
+        {
+          name: 'totalBonusPoints',
+          value: 'number',
+          description: 'Total bonus points the measurement can contribute',
+          notes: ''
+        },
+        {
+          name: 'noBenchmarks',
+          value: 'boolean',
+          description: 'If the measurement lacks benchmark data',
+          notes: ''
+        },
+        {
+          name: 'benchmarkType',
+          value: 'string',
+          description: 'Submission method with which the measurement benchmark data corresponds',
+          notes: ''
+        },
+        {
+          name: 'eMeasureId',
+          value: 'string',
+          description: 'The measurements eMeasureId',
+          notes: 'The eMeasureId from the measure\'s <a href="https://github.com/CMSgov/qpp-measures-data/blob/master/measures/measures-data.json">qpp-measures-data</a> definition'
+        },
+        {
+          name: 'cpcPlusGroup',
+          value: 'string char',
+          description: 'CPC+ group identifier for the measurement if it belongs to a CPC+ eligible measurement set',
+          notes: ''
+        },
+        {
+          name: 'messages',
+          value: 'string',
+          description: '',
+          notes: ''
+        }
+      ],
+      messages: [
+        {
+          name: 'measurementClass',
+          value: 'string',
+          description: 'Details of how the measurement class was determined',
+          notes: ''
+        },
+        {
+          name: 'decileScore',
+          value: 'string',
+          description: 'Details how the decile score was applied according to the classification of the measurement',
+          notes: ''
+        },
+        {
+          name: 'measurementPicker',
+          value: 'string',
+          description: 'The place in which the measurement was picked for scoring',
+          notes: ''
+        },
+        {
+          name: 'totalMeasurementPoints',
+          value: 'string',
+          description: 'How points are included in the total score for the measurement based on its processing status',
+          notes: ''
+        }
+      ]
+    }
+  }
+};
+
+const reformattedCode = (code) => code
+  .split('\n')
+  .map(l => l.trim())
+  .map((line, idx, arr) => {
+    if (idx > 0 && idx < arr.length - 1) {
+      return `  ${line}`;
+    }
+
+    return line;
+  })
+  .join('\n');
+
+const CodeBlock = ({json, xml}) => {
+  const reformattedJSON = reformattedCode(json);
+  // Need some extra processing to get hyperlinks to work without processing XML as JSX
+  const reformattedXML = reformattedCode(xml.replace(/<(?!a|\/a)/g, '&lt;').replace(/(!<\/a)>/g, '&gt;'));
```

Comment:
I made this more clear. I do a little more processing to set < and > into their HTML character entity references, because React will process XML as JSX inside of 'dangerouslySetInnerHTML' in <pre></pre> tags. This is one solution to escape that behavior.